watchman/checks.py

```diff
@@ -67,6 +73,14 @@ def _check_storage():
     return {"ok": True}
 
 
+@check
+def _check_amqp_connection(connection_url):
+    conn_params = pika.URLParameters(connection_url)
+    connection = pika.BlockingConnection(conn_params)
+    connection.close()
+    return {'ok': True}
```

Comment:
+1 - if this supports multiple AMQP connections, there should be some way of telling them apart.